src/harness/testcases/testdata/esc_sensor_record_0.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "esc_sensor/admin_0/esc_sensor_0",
+ "installationParam":{
+   "latitude": 39.0119,
+    "longitude": -98.4842,
+    "height": 9.3,
+    "heightType": "AGL",
+    "indoorDeployment": false,
```

Comment:
Defining a separate schema will be annoying to maintain but I don't see any better way. Thanks for doing this.